src/Persistence/Mapping/Driver/FileDriver.php

```diff
@@ -139,7 +139,7 @@ public function getAllClassNames(): array
      * @return ClassMetadata[]
      * @psalm-return array<class-string, ClassMetadata<object>>
      */
-    abstract protected function loadMappingFile(string $file): array;
+    abstract protected function loadMappingFile(string $file);
```

Comment:
does the ORM need to return something else than an array here ?

Reply:
No, it's just that the return type declaration was not added in ORM 3 (it was forgotten)